force-app/main/default/classes/TokenMigrationBatch.cls

```diff
@@ -104,47 +102,25 @@ global with sharing class TokenMigrationBatch implements Database.Batchable<sObj
                         continue;
                     }
                 }
-
+                
                 //exporting spreedly payment method token to stripe
                 FDService.EPayRequest req = SpreedlyEPayService.createEpayRequest(stipeGatewayToken, paymentMethod.OrderApi__Payment_Method_Token__c);
                 SpreedlyEPayService.createExportTokenBody(req);
                 FDService.EPayResponse epayResponse = SpreedlyHTTPService.prepareAndDoAPICallout(req);
                 
                 if (!epayResponse.transactions.isEmpty()) {
                     FDService.EPayTransaction trans = epayResponse.transactions[0];
-
-                    if (!contactsToUpdate.containsKey(contactId)) {
-                        //updating payment customer id field on contact
-                        contactsToUpdate.put(contactId, new Contact(Id=contactId, Payments_Customer_Id__c=trans.token));
-                    }
-
-                    OrderApi__Payment_Method__c newPaymentMethod = createPaymentMethod(trans, paymentMethod, contactId, accountId);
-                    newPaymentMethod.Original_Saved_Payment_Method__c = paymentMethod.Id;
-                    paymentMethodsToInsert.add(newPaymentMethod);
-
-                    //updating existing payment method active field as false
-                    paymentMethodsToUpdate.add(new OrderApi__Payment_Method__c(Id=paymentMethod.Id, OrderApi__Is_Active__c=false));
+                    paymentMethodsToUpdate.add(new OrderApi__Payment_Method__c(Id=paymentMethod.Id, Token_Migration_Test__c = 'SUCCESS'));
                 } else if (!epayResponse.errors.isEmpty()) {
-                    errorSystemLogs.add(SpreedlyHTTPService.createSystemLog('Spreedly Error Occured while Callout', epayResponse.errors[0].getMessage()));
+                    paymentMethodsToUpdate.add(new OrderApi__Payment_Method__c(Id=paymentMethod.Id, Token_Migration_Test__c = 'Error: ' + epayResponse.errors[0].getMessage()));
                 }
             }
-
-            //Updating contact records
-            if (!contactsToUpdate.isEmpty()) {
-                Framework.SObjectService.updateRecords(contactsToUpdate.values());
-            }
-
             if (!paymentMethodsToUpdate.isEmpty()) {
                 Framework.SObjectService.updateRecords(paymentMethodsToUpdate);
             }
-
-            if (!paymentMethodsToInsert.isEmpty()) {
-                Framework.SObjectService.insertRecords(paymentMethodsToInsert);
-            }
-
             SpreedlyHTTPService.insertSystemLogs();
         } catch(Exception e) {
-            errorSystemLogs.add(SpreedlyHTTPService.createSystemLog('Spreedly Error Occured', e.getMessage() + ' ' + e.getStackTraceString()));
+            errorSystemLogs.add(SpreedlyHTTPService.createSystemLog('Spreedly Error Occure', e.getMessage() + ' ' + e.getStackTraceString()));
```

Comment:
@jupadhyaya Please correct the spelling of Occured!

Reply:
updated